src/scrapers/max.ts

```diff
@@ -124,11 +127,11 @@ function getTransactionType(txnTypeStr: string) {
 
 function getInstallmentsInfo(comments: string) {
   if (!comments) {
-    return null;
```

Comment:
Just wondering why you prefer to return undefined instead of null?

Reply:
`getInstallmentsInfo` was called with `|| undefined` which is basically converts null to undefined - so it's more straight forward to just return `undefined`